### DISCUSSION TITLE
Open a note via link without expanding the parent note in the note tree

### DISCUSSION THREAD

**Discussion by XaverStiensmeier** [TriliumNext/Trilium#4660] [Q&A]
As I describe in this discussion: https://github.com/zadam/trilium/discussions/4634 I have written a script to import a few hundred notes into a single note - lets call that note "parentNote". They are only used to be linked to in other notes.

Whenever I click on a link to open one of these notes, the parentNote is expanded in the tree view. That leads to me needing to scroll all the way up and collapse it again every time.

Is there a way how you can prevent the note from expanding? It would be okay if it would never expand at all (meaning if it would be impossible to see that note in the treeview for example which would limit opening such notes to links).

If an example helps why such behaviour might be of interest: Think of a lexicon. If you link a word you do not want to scroll through all words in a lexicon to close the lexicon to get back to business (I am aware of the backlink, but the parentNote remains expanded making future orientation less simple.

**Comment by rom1dep** [comment]
Wouldn't it be nice to have sticky tree parents in the notes tree like so: https://marchaos.github.io/react-virtualized-sticky-tree/ ?
Would that solve your issue? If so, I wonder if it can be achieves with some CSS custom script https://stackoverflow.com/questions/63421400/sticky-rows-in-tree-like-nested-list-structure

**Reply by XaverStiensmeier** [reply]
It wouldn't be a solve solve given that the parentNote would still expand, but it would be a feasible work around if I can click the still parentNote while it being sticky.

**Comment by zadam** [comment]
I experimented a bit and found a bit of a hack. There's a concept of archived notes: https://github.com/zadam/trilium/wiki/Archived-notes

With a specific setup you can achieve what you want:

* set the parent "Lexicon" as archived, but not the children (this will allow them to stay e.g. in the linking dialog)
* set "Hide archived notes" to true:

![image](https://github.com/zadam/trilium/assets/617641/f2d44abc-528c-4e46-b394-e34bd64aa8af)

It has disadvantages - e.g. to add new items to the lexicon, you'd have to toggle the "Hide archived notes" to false.
